include/dali/core/tensor_shape.h

```diff
@@ -244,11 +244,19 @@ struct TensorShape<DynamicDimensions>
    * @brief Return empty shape of specified dimensionality
    */
   static TensorShape<> empty_shape(int dim) {
+    return TensorShape<>::filled_shape(dim, 0);
+  }
+
+  /**
+   * @brief Return shape of specified dimensionality filled with value
+   */
+  static TensorShape<> filled_shape(int dim, int64_t value) {
     assert(dim > 0);
     TensorShape<> result;
     result.resize(dim);
+    // TODO: should use std::fill but lack .begin() and .end()
```

Comment:
Please remove this comment. Firstly, `TensorShape` _does have_ `begin` and `end`, as clearly indicated by the following  ranged for. Secondly, at least in my opinion, assigning elements in a ranged `for` reads better than the very verbose call to std::fill (STL ranges change that but we're not there yet with the standard support).

Reply:
My bad, LSP complained it didn't exist and I wasn't thinking carefully (classic 1am moment).
I plan to play with trying to compile with cxx_std_23 later.